include/swift/AST/Module.h

```diff
@@ -373,8 +373,9 @@ class ModuleDecl : public DeclContext, public TypeDecl {
   void
   getImportedModulesForLookup(SmallVectorImpl<ImportedModule> &imports) const;
 
-  /// Extension of the above hack. Identical to getImportedModulesForLookup()
-  /// for imported modules, otherwise also includes @usableFromInline imports.
+  /// Extension of the above hack. Behaves identically to
+  /// getImportedModulesForLookup() for Clang modules, otherwise also
```

Comment:
I still think you shouldn't mention the Clang part at all. Why does someone care about that when using the API?